docs: Add missing audit logs filtering fields (#4133)
coder/coder: docs/admin/audit-logs.md

The supported filters are:

- `resource_type` - The type of the resource. It can be a workspace, template, user, etc. You can [find here](https://pkg.go.dev/github.com/coder/coder@main/codersdk#ResourceType) all the resource types that are supported.
- `resource_id` - The ID of the resource.

> - `resource_target` - The name of the resource. It is useful when you don't know the ID of a resource but know its name.

Reviewer: ```suggestion
- `resource_target` - The name of the resource. Can be used instead of `resource_id`.
```
